composer.json

```diff
@@ -18,30 +18,29 @@
   "require" : {
     "php" : ">=8.1",
     "ext-json": "*",
-    "thecodingmachine/graphqlite" : "^6.0",
-    "thecodingmachine/graphqlite-symfony-validator-bridge" : "^6.0",
-    "symfony/framework-bundle": "^6",
-    "symfony/validator": "^6",
-    "symfony/translation": "^6",
+    "thecodingmachine/graphqlite" : "^7.0",
+    "thecodingmachine/graphqlite-symfony-validator-bridge": "dev-master",
+    "symfony/framework-bundle": "^6.0 || ^7.0",
+    "symfony/validator": "^6.0 || ^7.0",
+    "symfony/translation": "^6.0 || ^7.0",
     "doctrine/annotations": "^1.13 || ^2.0.1",
-    "symfony/psr-http-message-bridge": "^2.0",
+    "symfony/psr-http-message-bridge": "^6.0 || ^7.0",
     "nyholm/psr7": "^1.1",
-    "laminas/laminas-diactoros": "^2.2.2",
+    "laminas/laminas-diactoros": "^2.2 || ^3.0",
     "overblog/graphiql-bundle": "^0.2 || ^0.3",
```

Comment:
This package is not v7 ready. How did you overcome this?

Reply:
I had to add it into `conflicts` in order to use v7 symfony

Reply:
other bundles may still conflict with symfony v7, for example some components are blocked by `overblog/graphiql-bundle`, this PR's goal is to not be blocker of symfony v7 migration, if laminas will enable support for v7 in next major version I will create following PR to add support of it.